Compiler "assertion failed" when using unknown type

### Expected behavior
Get a clear error that the type is unknown. Possibly other related errors when using the type.

### Steps to reproduce

Compile the following code:
```typescript
class MyClass {
  get value(): UnknownType {
  }
}

export function test(obj: MyClass): void {
  obj.value;
}
```

On the playground https://www.assemblyscript.org/editor.html I get the "assertion failed" in the browser's console:

```
AssertionError: assertion failed
    at i.assert (https://cdn.jsdelivr.net/npm/assemblyscript@0.18.12/dist/assemblyscript.js:7:722100)
    at f.compilePropertyAccessExpression (https://cdn.jsdelivr.net/npm/assemblyscript@0.18.12/dist/assemblyscript.js:7:317653)
    at f.compileExpression (https://cdn.jsdelivr.net/npm/assemblyscript@0.18.12/dist/assemblyscript.js:7:236891)
    at f.compileExpressionStatement (https://cdn.jsdelivr.net/npm/assemblyscript@0.18.12/dist/assemblyscript.js:7:225012)
    at f.compileStatement (https://cdn.jsdelivr.net/npm/assemblyscript@0.18.12/dist/assemblyscript.js:7:221342)
    at f.compileStatements (https://cdn.jsdelivr.net/npm/assemblyscript@0.18.12/dist/assemblyscript.js:7:222370)
    at f.compileFunctionBody (https://cdn.jsdelivr.net/npm/assemblyscript@0.18.12/dist/assemblyscript.js:7:211588)
    at f.compileFunction (https://cdn.jsdelivr.net/npm/assemblyscript@0.18.12/dist/assemblyscript.js:7:210572)
    at f.compileElement (https://cdn.jsdelivr.net/npm/assemblyscript@0.18.12/dist/assemblyscript.js:7:202495)
    at f.compileExports (https://cdn.jsdelivr.net/npm/assemblyscript@0.18.12/dist/assemblyscript.js:7:203235)
```

--- a/src/compiler.ts
+++ b/src/compiler.ts
@@ -9019,7 +9019,8 @@ export class Compiler extends DiagnosticEmitter {
       }
       case ElementKind.PROPERTY: {
         let propertyInstance = <Property>target;
-        let getterInstance = assert(propertyInstance.getterInstance);
+        let getterInstance = propertyInstance.getterInstance;
+        if (!getterInstance) return module.unreachable(); // failed earlier
         let thisArg: ExpressionRef = 0;
         if (getterInstance.is(CommonFlags.INSTANCE)) {
           thisArg = this.compileExpression(

